.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "sample"]
+	path = recursive-git-log-plugin/src/test/resources/multi-project-support-sample
+	url = git@github.com:line/multi-project-support-sample.git
```

Comment:
This remote repo is still `private`.

Let me share when it switch to `public`.